### PR TITLE
Added player team color now stored as variable

### PR DESCRIPTION
**Pull request by ColdEvul**
**When merged this pull request will:**
- Added player team color stored as variable 

Can be usefull for scripting. I intend to use it to ensure color is used on respawn for instance

### CODE REVIEW

File: addons/interaction/functions/fnc_joinTeam.sqf [hunk 1]
```diff
@@ -19,6 +19,7 @@
 params ["_unit", "_team"];
 
 ["CBA_teamColorChanged", [_unit, _team]] call CBA_fnc_globalEvent;
+SETVAR(_unit, QGVAR(player_team), _team);
```
Comment (jonpas) [inline addons/interaction/functions/fnc_joinTeam.sqf]: We don't really use that macro anymore. I'd suggest:
```suggestion
_unit setVariable [QGVAR(playerTeam), _team];
```

**Comment by commy2** [comment]
Literally just https://community.bistudio.com/wiki/assignedTeam

I refuse having this as a work around, even if BI broke their shit for no reason.

**Comment by jonpas** [comment]
> I refuse having this as a work around, even if BI broke their shit for no reason.

If the command is broken, why not provide a working solution? We can say the same about many other ACE and CBA systems.

**Comment by commy2** [comment]
But this is so stupid it hurts. I don't want to take part in this embarrassment.

**Comment by jonpas** [comment]
Add a comment what it's for.

**Comment by PabstMirror** [comment]
Could also use the `"CBA_teamColorChanged"` event to set variables you need

**Comment by ColdEvul** [comment]
I use a similar variable in our script package to make sure the team is reapplied on death.

I figured I just could export it to this function. Nothing serius or any game changing you just have a player team variable to play with.

Using the last team color changed to.

**Comment by jonpas** [comment]
Hmm, it's a 1-liner to do what @PabstMirror said, that's not that hard for the few mods that want to use that.

**Comment by ColdEvul** [comment]
This one can be closed then. :)

**Pull request closed without merging**